### PR TITLE
polish(chainwatch): accept lotus api and token

### PR DESCRIPTION
**Pull request by frrist**
Completes https://github.com/filecoin-project/sentinel/issues/47. Will leave draft once manually validated.

### CODE REVIEW

File: cmd/lotus-chainwatch/main.go [hunk 1]
```diff
@@ -26,6 +26,11 @@ func main() {
 				EnvVars: []string{"LOTUS_PATH"},
 				Value:   "~/.lotus", // TODO: Consider XDG_DATA_HOME
 			},
+			&cli.StringFlag{
+				Name:    "api",
+				EnvVars: []string{"FULLNODE_API_INFO"},
```
Comment (placer14) [inline cmd/lotus-chainwatch/main.go:31]: Missing `LOTUS_API_INFO`.

File: cmd/lotus-chainwatch/util/api.go [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+import (
+	"context"
+	"net/http"
+
+	"github.com/filecoin-project/go-jsonrpc"
+	"github.com/filecoin-project/lotus/api"
+	"github.com/filecoin-project/lotus/api/client"
+	ma "github.com/multiformats/go-multiaddr"
+	manet "github.com/multiformats/go-multiaddr-net"
```
Comment (placer14) [inline cmd/lotus-chainwatch/util/api.go:11]: Linter failure:
```
cmd/lotus-chainwatch/util/api.go:20:18: SA1019: manet.DialArgs is deprecated: use github.com/multiformats/go-multiaddr/net  (staticcheck)
	_, addr, err := manet.DialArgs(parsedAddr)
	                ^
cmd/lotus-chainwatch/util/api.go:11:2: SA1019: package github.com/multiformats/go-multiaddr-net is deprecated: use github.com/multiformats/go-multiaddr/net  (staticcheck)
	manet "github.com/multiformats/go-multiaddr-net"
	^
```

File: cmd/lotus-chainwatch/main.go (hunk 1 above)
Comment (frrist) [reply cmd/lotus-chainwatch/main.go:31]: Per out of band discussion we are going to drop the requirement that there be 2 env vars for this, we will just look for `FULLNODE_API_INFO`

File: cmd/lotus-chainwatch/util/api.go (hunk 2 above)
Comment (frrist) [reply cmd/lotus-chainwatch/util/api.go:11]: derp, will address.

**Review by magik6k — APPROVED** [review]

**Pull request merged**